### DISCUSSION TITLE
Improved Kratach (ALGEROTH) [U]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#54] [Card Balance]
Improved Kratach (ALGEROTH) [U]
EQUIPMENT, FIREARM
[S] +5.

My suggestion : 
EQUIPMENT, FIREARM
[S] +5.Equipped warrior may be attached EQUIPMENT during your turn for no Action cost.

Works with :
-Azogar Polearm
-Bacteria grenade
-Black bullets
-Carcass suits 🙃
-Chain Bayonet
-The black gate
-Coagulant autoinjector

**Comment by Sol-au-fein** [comment]
I dont know how an Improved Kratach is superior to Nimrod autocannon, heaviest machine gun available to humankind. I suggest to rework Improved Kratach with some ability, instead of high modifier.
In paper doomtrooper, there is also standard Kratach, which is +3, which could have no atachment cost, since it is basicaly equivalent of AK-47.
